integrations/cohere/src/haystack_integrations/components/rankers/cohere/ranker.py

```diff
@@ -152,7 +166,7 @@ def run(self, query: str, documents: List[Document], top_k: Optional[int] = None
             model=self.model_name,
             query=query,
             documents=cohere_input_docs,
-            max_chunks_per_doc=self.max_chunks_per_doc,
+            max_tokens_per_doc=self.max_tokens_per_doc,
```

Comment:
I would suggest to set at the same default value as the API sets if nothing is given.

```suggestion
            max_tokens_per_doc=self.max_tokens_per_doc or 4096,
```